recipes/s3contents/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "s3contents" %}
+{% set version = "0.2.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "ab8904b0e4881dd3be0e030e39035d9f2ea45e162bb8bee88faf576df9897e47" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  # skipped on windows due to an issue in the library itself
+  skip: True  # [win]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - jupyter
+    - boto3
+    - requests
+    - s3fs
+    - gcsfs
+
+test:
+  requires:
```

Comment:
I don't think you need `nose` and `mock` to import the module. You could probably also skip testing the import of `s3contents.tests` since it basically no-ops.

Reply:
You actually do need them unfortunately :( .  Planning to fix that upstream.